### PR TITLE
chore: removed travis yml and added git action support

### PR DESCRIPTION
**Pull request by mnoman09**
Summary
-------
-  We remove travis yaml from python sdk and moved all the stages to git action.

Test plan
---------
All tests should pass

https://github.com/optimizely/python-sdk/runs/6184738570?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    secrets:
+      CI_USER_TOKEN:
+        required: true
+      TRAVIS_COM_TOKEN:
+        required: true
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```
Comment (msohailhussain) [inline .github/workflows/integration_test.yml:18]: use @v3

File: .github/workflows/python.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        python-version: 3.9
+      # flake8 version should be same as the version in requirements/test.txt
+      # to avoid lint errors on CI
+    - name: pip install flak8
+      run: pip install flake8>=4.1.0
+    - name: Lint with flake8
+      run: |
+        flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```
Comment (msohailhussain) [inline .github/workflows/python.yml:44]: i don't see this line in travis?

File: .github/workflows/python.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```
Comment (msohailhussain) [inline .github/workflows/python.yml:46]: what does it mean?

**Review by msohailhussain — APPROVED** [review]
please address comments and lgtm

File: .github/workflows/python.yml (hunk 3 above)
Comment (mnoman09) [reply .github/workflows/python.yml:46]: Comments are written above for clarification. These lines were generated automatically so I didn't removed it as it seemed useful

File: .github/workflows/integration_test.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        # You should create a personal access token and store it in your repository
+        token: ${{ secrets.CI_USER_TOKEN }}
+        repository: 'optimizely/travisci-tools'
+        path: 'home/runner/travisci-tools'
+        ref: 'master'
+    - name: set SDK Branch if PR
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "SDK_BRANCH=${{ github.head_ref }}" >> $GITHUB_ENV
+        echo "TRAVIS_BRANCH=${{ github.head_ref }}" >> $GITHUB_ENV
```
Comment (msohailhussain) [inline .github/workflows/integration_test.yml:29]: remove it.

Comment (msohailhussain) [inline .github/workflows/integration_test.yml:29]:
```suggestion
```

File: .github/workflows/source_clear_cron.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+on:
+  schedule:
+    # Runs "weekly"
+    - cron: '0 0 * * 0'
+
+jobs:
+  source_clear:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (msohailhussain) [inline .github/workflows/source_clear_cron.yml:12]:
```suggestion
      - uses: actions/checkout@v3
```

File: .github/workflows/python.yml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        
+  integration_tests:
+    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@master
```
Comment (msohailhussain) [inline .github/workflows/python.yml:49]:
```suggestion
    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@mnoman/gitAction
```

File: .github/workflows/python.yml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    secrets:
+      CI_USER_TOKEN: ${{ secrets.CI_USER_TOKEN }}
+      TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
+      
+  fullstack_production_suite:
+    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@master
```
Comment (msohailhussain) [inline .github/workflows/python.yml:55]:
```suggestion
    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@mnoman/gitAction
```

File: .github/workflows/python.yml [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    - name: Lint with flake8
+      run: |
+        flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        
+  integration_tests:
+    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@mnoman/gitAction
```
Comment (msohailhussain) [inline .github/workflows/python.yml:49]:
```suggestion
    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@master
```

File: .github/workflows/python.yml [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+    secrets:
+      CI_USER_TOKEN: ${{ secrets.CI_USER_TOKEN }}
+      TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
+      
+  fullstack_production_suite:
+    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@mnoman/gitAction
```
Comment (msohailhussain) [inline .github/workflows/python.yml:55]:
```suggestion
    uses: optimizely/python-sdk/.github/workflows/integration_test.yml@master
```

**Pull request merged**